docs/earthpy-data-subsets.rst

```diff
@@ -0,0 +1,83 @@
+=====================
+EarthPy Data Subsets
+=====================
+
+EarthPy comes with the ability to download example datasets using
+``earthpy.data.get_data()``. These datasets are used in example documentation
+on how to use EarthPy, as well as in the EarthPy related lessons on
+earthdatascience.org. There are many keys that can be used to get data
```

Comment:
It would be nice for earthdatascience.org to have a hyperlink, and a brief description after the word like:
"earthdatascience.org, the open learning portal developed and maintained by Earth Lab at CU Boulder". 

Rather than "There are many keys..", I would be more explicit: "To download these datasets, you can provide the appropriate data key to the ```earthpy.data.get.data() function``` for one of the datasets listed below (e.g.. ```earthpy.data.get_data("colorado-flood")```).